backend/src/main/java/net/wodie/backend/service/BtoService.java

```diff
@@ -16,10 +17,16 @@ public BtoService(BtoRepository btoRepository) {
 
     public List<BtoItem> getAllBtoItems() {
         return btoRepository.findAll();
-
     }
 
     public BtoItem updateBtoItem(BtoItem updatedBtoItem) {
         return btoRepository.save(updatedBtoItem);
     }
+
+    public String getBtoItemStatusById(String id) {
+        return  btoRepository.findById(id)
+                .orElseThrow( () -> new NoSuchElementException("BtoItem not found with id: " + id))
+                .getStatus();
+
```

Comment:
getStatus() brauchst du den da oder hat der sich da eingeschlichen

Reply:
Hallo Tim,
danke für Deinen Hinweis, der Eintrag hier wird benötigt, da ich in der Funktion getBtoItemStatusById zuerst auf eine fehlerhafte Rückgabe prüfen muß und dann erst auf des Feld Status zugreifen kann.